app/tour_step_directive.js

```diff
@@ -91,7 +91,8 @@
 
                     //Add step to tour
                     templateReady.then(function () {
-                        ctrl.addStep(step);
+						ctrl.addStep(step);
+						scope.step = step;
```

Comment:
Done!

2015-07-12 22:26 GMT+02:00 benmarch <notifications@github.com>:

> In app/tour_step_directive.js
> <https://github.com/benmarch/angular-bootstrap-tour/pull/24#discussion_r34425256>
> :
>
> > @@ -91,7 +91,8 @@
> >
> >                      //Add step to tour
> >                      templateReady.then(function () {
> > -                        ctrl.addStep(step);
> > +						ctrl.addStep(step);
> > +						scope.step = step;
>
> Can you change this to "scope.tourStep"? I think "step" will be too
> general and might override other scope properties. It looks like you are
> using tabs instead of spaces, can you replace with 4 spaces, please? I will
> merge after these changes are made.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benmarch/angular-bootstrap-tour/pull/24/files#r34425256>
> .
>



-- 
There are 10 kinds of people. Those that understand binary notation and
those that don't.